pos_order_mgmt/static/src/js/widgets.js

```diff
@@ -210,6 +210,17 @@ odoo.define('pos_order_mgmt.widgets', function (require) {
             order.trigger('change');
             this.pos.get('orders').add(order);
             this.pos.set('selectedOrder', order);
+            order.get_orderlines().forEach(function (orderline) {
+                if (orderline.pack_lot_lines && orderline.return_lot_no)
+                {
+                    var PacklotlineCollection = pos.Packlotline;
+                    for (var i = 0; i < orderline.return_lot_no.split(',').length; i++) {
+                        orderline.pack_lot_lines.add(new PacklotlineCollection({'lot_name': orderline.return_lot_no.split(',')[i]}, {'order_line': orderline}));
+                    }
```

Comment:
I'm hoping this method would get a lot cleaner if you send the list, instead of concatenating.. because you're splitting multiple times inside the loop and that's just not clean

Reply:
If you're able to send the list, which I think should be possible because it's valid json, then hre you can probalby use an underscore helper function to loop

something like
```js
_.each(orderline.return_pack_lot_names, function(lot_name) { orderline.pack_lot_lines.add(new PacklotlineCollection({'lot_name': lot_name, ... })
```